go-controller/pkg/ovn/dns_name_resolver/dns.go

```diff
@@ -0,0 +1,255 @@
+package dnsnameresolver
```

Comment:
I think you left the old `ovn/egressfirewall_dns` file, that is why it sees this as new and not moved

Reply:
I have fixed the issue.